Topic4/src/classes/MarketManager.java

```diff
@@ -0,0 +1,34 @@
+package classes;
+
+import interfaces.Manager;
+
+public class MarketManager implements Manager {

```

Comment:
The managers to be notified should have at least an email address to allow the delivery of the different type of notifications